project.clj

```diff
@@ -10,12 +10,12 @@
                  [ring/ring-defaults "0.2.1"]
                  [ring/ring-json "0.4.0"]
                  [ring/ring-jetty-adapter "1.5.0"]
-                 [environ "1.0.0"]]
+                 [environ "1.1.0"]]
   :min-lein-version "2.0.0"
   :plugins [[lein-ring "0.9.7"]
-            [environ/environ.lein "0.3.1"]]
```

Comment:
Note: could be a dependency for deploying on heroku according to heroku's clojure example app:
https://github.com/heroku/clojure-getting-started/blob/master/project.clj